app/src/components/market/common/transaction_details_card/index.tsx

```diff
@@ -2,8 +2,11 @@ import React from 'react'
 import styled from 'styled-components'
 
 const Wrapper = styled.div`
-  border: 1px solid ${props => props.theme.borders.borderColor};
   padding: 21px;
+  position: relative;
+  background-image: url(/svgs/note-border-big.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
```

Comment:
We should really be creating the zigzag line in css instead of using an svg as it can result in all kinds of bugs. For example, if you change the currency, the left border suddenly disappears: 
![image](https://user-images.githubusercontent.com/30579067/95612652-97f76d80-0a18-11eb-8a4c-580ccd085f2f.png)

Instead, this codepen should serve as a good reference as to how to get zigzag line in css only: https://codepen.io/zoerooney/pen/pvOaxR

Reply:
![Screen Shot 2020-10-11 at 10 43 55 PM](https://user-images.githubusercontent.com/34900782/95699742-5383f180-0c13-11eb-8103-c62637f94b12.png)

Hi @KadenZipfel  @pimato , I followed that codepen sample and did it.
But it's difficult to match the corners